src/SourceCode.Clay.OpenApi/Expressions/OasExpression.cs

```diff
@@ -86,18 +86,12 @@ public bool Equals(OasExpression other)
         /// <returns>A 32-bit signed integer that is the hash code for this instance.</returns>
         public override int GetHashCode()
         {
-            unchecked
-            {
-                var hc = 17L;
+            var hc = new HashCode();
 
-                if (_components != null)
-                {
-                    hc = (hc * 23) + _components.Length;
-                    hc = (hc * 23) + _components[0].GetHashCode();
-                }
+            if (_components != null)
+                hc = hc.TallyCount(_components).Tally(_components[0]);
```

Comment:
Nice

Reply:
But I hope it does in fact inline. Because there's a lot more stack frames here for logically the same result.